src/pages/signin/SignInHeroCopy.js

```diff
@@ -9,10 +10,16 @@ import styles from '../../styles/styles';
 import variables from '../../styles/variables';
 
 const propTypes = {
+    /** Override the green headline copy */
+    customHeadline: PropTypes.string,
```

Comment:
NAB. I think this should be `headline` or `headlineText` and marked as required.

The better alternative is to just make the default value `props.translate('login.hero.header')` but I don't think this is possible, we can't make the default value of one prop dependent on another prop.

Reply:
> but I don't think this is possible, we can't make the default value of one prop dependent on another prop.
Yeah this was my thought as well- I agree it would be better as required and then just pass the headline function as an option to the react navigator stack 
I think for now, we can leave this as is, as it'll only be used for conferences and creating custom sign in pages for events such as in this PR

Reply:
To be honest, I don't think that this change is worth blocking the PR since this is a critical issue

Reply:
I'll add a link to this comment in the description of this PR for the next time we update the demos?

Reply:
Sounds good to me 👍